package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pangolindex/components",
-  "version": "2.0.8",
+  "version": "2.0.3",
```

Comment:
please don't update this version, this is automatic updation

Reply:
Yeah I am sorry, I tought that I should keep the version of the PR, so you want me to put 2.0.8? Or should I leave it as .3 and it will update?

Reply:
yes 2.0.8 is fine, thanks a lot